ui/statistics/src/main/java/de/danoeh/antennapod/ui/statistics/years/BarChartView.java

```diff
@@ -111,14 +113,21 @@ public void draw(@NonNull Canvas canvas) {
             float maxLine = (float) (Math.floor(maxValue / (10.0 * ONE_HOUR)) * 10 * ONE_HOUR);
             float y = (1 - (maxLine / maxValue)) * barHeight;
             canvas.drawLine(0, y, width, y, paintGridLines);
-            canvas.drawText(String.valueOf((long) maxLine / ONE_HOUR), 0, y + 1.2f * textSize, paintGridText);
+            if (maxLine > 0) {
```

Comment:
Is it a problem to draw this line at 0? Leaving this as it is would keep the code a bit easier.